### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by grosser**

### CODE REVIEW

**Comment by Skarlso** [comment]
Just saying that this is not a good practice. 

```
File.read("#{config}/foo.yml")
```

Since it leaves open the file descriptor. The shortest is 

```
File.open('bla', 'r', &:read).chop
```

It passes in a block which will close the descriptor at the end.

**Comment by grosser** [comment]
"Opens the file, optionally seeks to the given offset, then returns
length bytes
(defaulting to the rest of the file). read ensures the file is closed
before returning."

On Tue, Oct 4, 2016 at 10:29 PM, Gergely Brautigam <notifications@github.com

**Comment by Skarlso** [comment]
@grosser Damn it. :O I was looking for this and couldn't find it. :D Well done. I stand corrected. Also, I learned something today, which is nice. :) Thanks 👍

File: README.md [hunk 1]
```diff
@@ -95,44 +96,25 @@ end
...
+Clone existing directories or files to reuse them during tests, they are safe to modify.
 
 ```ruby
 FakeFS do
-  FileUtils.mkdir_p(File.join(__dir__, '../cfg'))
-  File.open(File.join(__dir__, '../cfg/config.yml'), 'w') do |f|
-    f.puts('loglevel: OVERWRITTEN_VALUE')
-    f.puts('someotherthingmore: fakedata')
-  end
+  config = File.expand_path('../../config', __FILE__)
```
Comment (Skarlso) [inline README.md]: Good Stuff, I'm going to use expand_path as well. :)

**Comment by grosser** [comment]
Just a silly example ... but it actually works ;)
YAML.load("NEW") == "NEW"

**Comment by Skarlso** [comment]
Bah, I give up! :D

**Comment by grosser** [comment]
💃

**Pull request merged**